### DISCUSSION TITLE
sdfat

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D#819] [Questions]
hi luc, the code of  new version  did not support sdfat, only support sdfat2。
can you add the sdfat function？

**Comment by luc-github** [comment]
why ? https://github.com/greiman/SdFat/tree/2.1.2 is better than https://github.com/greiman/SdFat/tree/1.1.4

Why need to support old library ?

**Comment by liqijian101** [comment]
when I use esp8266，the upload speed of sdfat is more faster than sdfat2

**Reply by luc-github** [reply]
it may be library configuration issue, with esp32 is faster, did you try native sd with esp8366 ? it also use sdfat
